documents/knowledge/elements.yaml

```diff
@@ -3179,10 +3179,411 @@
+  arity: 1
+  overloads:
+    lst: Center(a) (Pad each item with spaces so all are centered)
+  vectorise: false
+  tests:
+    - '[["ab","cdef"]] : [" ab ","cdef"]'
+    - '[["xyz","a","bcdef"]] : [" xyz ","  a  ","bcdef"]'
+
+- element: "øe"
+  name: Run Length Encoding
+  arity: 1
+  description: Run length encoding, convert from string to list of characters and amount repeated.
+  overloads:
+    str: run_length_encoded(a)
+  vectorise: false
+  tests:
+    - '["abc"] : [["a",1],["b",1],["c",1]]'
+    - '["aaa"] : [["a",3]]'
+
+- element: "ød"
+  name: Run Length Decoding
+  arity: 1
+  description: Run length decoding, convert from list of characters and lengths to a string
+  overloads:
+    lst: run_length_decoded(a)
+  vectorise: false
+  tests:
+    - '[[["x",3]]] : "xxx"'
+    - '[[["z",2],["a",3]]] : "zzaaa"'
+
+- element: "øD"
+  name: Dictionary Compression
+  description: Optimally compress a string of English using words from Vyxal's dictionary
+  arity: 1
+  overloads:
+    str: dictionary_compressed(a)
+  vectorise: false
+  tests:
+    - '["withree"] : "`wi∧ḭ`"'
+    - '["hello"] : "`ƈṙ`"'
+    - '["Vyxal"] : "`₴ŀ`"'
+
+- element: "øW"
+  name: Group on words
+  description: Group a string on words
+  arity: 1
+  overloads: 
+    str: Group a on words, leaving chunks of [a-zA-Z] together and having everything else as a single character
+  vectorise: false
+  tests:
+    - '["abc*xyz"] : ["abc","*","xyz"]'
+    - '["$$$"] : ["$","$","$"]'
+
+- element: "øṙ"
+  name: Regex replace
+  description: Replace matches of a with c in b
+  arity: 3
+  overloads:
+    any-any-fun: Apply c to matches of a in b
+    any-any-any: Replace matches of a with c in b
+  vectorise: false
+  tests:
+    - '[".{3}","hello","x"] : "xlo"'
+    - '["\W","Hello, World!","E"] : "HelloEEWorldE"'
+
+- element: "øm"
+  name: Palindromise
+  description: palindromise without duplicating middle
+  arity: 1
+  overloads:
+    any: palindromise(a) (Doesn't duplicate middle)
+  vectorise: false
+  tests: 
+    - '[[1,2,3]] : [1,2,3,2,1]'
+    - '["hello"] : "hellolleh"'
+
+- element: "øp"
+  name: Starts With
+  arity: 2
+  description: Check if one value starts with another
+  overloads:
+    any-any: a.startswith(b) (Starts with b?)
+  vectorise: false
+  tests:
+    - '["hello","h"] : 1'
+    - '["hello","hello"] : 1'
+    - '["hello","x"] : 0'
+    - '["hello",""] : 1'
+
+- element: "øP"
+  name: Pluralise Count
+  description: Create a sentence of the form 'a bs'
+  arity: 2
+  overloads:
+    num-str: a + " " + b + (s if a != 1 else "") (Concatenate with space, append a s if not 1)
+  vectorise: false
+  tests:
+    - '[4,"hello"] : "4 hellos"'
+    - '[1,"hello"] : "1 hello"'
+    - '[0,"hello"] : "0 hellos"'
+
+- element: "øṁ"
+  name: Vertical Mirror
+  description: Vertical Mirror - Split by newlines, mirror each line, join by newlines
+  arity: 1
+  overloads:
+    str: vertical_mirror(a)
+  vectorise: true
+  tests:
+    # TODO: MULTILINE TESTCASSES
```

Comment:
YAML multiline strings can be done with `>-` if we want to turn the newlines into spaces or `|` if we want to keep the newlines.

Reply:
Ok, I'll do this in a bit.